### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by zo-shin**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -233,43 +233,43 @@
...
     <string name="conn_timeout">Connection timed out</string>
 
-    <string name="already_default">It already be set to default</string>
+    <string name="already_default">It has already been set to default</string>
 
-    <string name="set_to_default">It will be set to default after using it open link(%s), please choose via to open the link.</string>
+    <string name="set_to_default">It will be set to default after opening link(%s), please choose via to open the link.</string>
 
-    <string name="quick_back_desc">There may be compatibility issues when you open this switch. You should close it when you found going back work not well.</string>
+    <string name="quick_back_desc">There may be compatibility issues when you enable this option. You should disable it when you find going back not work well.</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Could you change this to (Or at least more like it) There may be compatibility issues when you enable this option. You should disable it if you find going back working poorly.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -183,9 +183,9 @@
     <string name="user">Log in/Sign up</string>
     <string name="user_syns">Sync</string>
     <string name="user_update">Upload</string>
-    <string name="log_in_success">Log in success.</string>
+    <string name="log_in_success">Log in successful.</string>
     <string name="wrong_psw">Wrong password.</string>
-    <string name="sign_up_success">Sign up success.</string>
+    <string name="sign_up_success">Sign up successful.</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Maybe you could add -ly to successful?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -183,9 +183,9 @@
     <string name="user">Log in/Sign up</string>
     <string name="user_syns">Sync</string>
     <string name="user_update">Upload</string>
-    <string name="log_in_success">Log in success.</string>
+    <string name="log_in_success">Log in successful.</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Maybe you could change this to Log in was successful.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -168,7 +168,7 @@
     <string name="desc_filedic">file</string>
     <string name="desc_other">other</string>
     <string name="msg_setgoogle">Set Google search engine now?</string>
-    <string name="action_mark">Mark as AD</string>
+    <string name="action_mark">Mark as Ad</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Maybe you could change this to Mark as an AD

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -114,25 +114,25 @@
     <string name="toast_save_to">Save file to: </string>
     <string name="toast_operation_failed">The operation failed.</string>
     <string name="toast_operation_done">The operation done.</string>
-    <string name="volume">Press Volumekey to Scroll</string>
-    <string name="intent">Allow Website Redirect to App</string>
-    <string name="msg_intent">Would like to open your app?</string>
-    <string name="dialog_adblock">Default Block:Use Local hosts.txt to Block Ads\n\nCustom Block:Use storage hosts.txt, it will download a file if unexisted.</string>
+    <string name="volume">Press Volume Keys to Scroll</string>
+    <string name="intent">Allow Website to Run Apps</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Maybe you could change this to Allow This Website to Open its App

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -97,14 +97,14 @@
...
-    <string name="image_off">Show image</string>
+    <string name="image_on">Block images</string>
+    <string name="image_off">Show images</string>
     <string name="title_file_chooser">File chooser</string>
     <string name="import_backup">Import bookmarks</string>
     <string name="bookmark_export_path">Bookmarks exported to\n</string>
     <string name="import_bookmark_error">Could not import bookmarks from file</string>
     <string name="feedback">Feedback</string>
-    <string name="message_add_bookmark">Added bookmark successfully</string>
+    <string name="message_add_bookmark">Bookmark added successfully</string>
```
Comment (Vv2233Bb) [inline app/src/main/res/values/strings.xml]: Could you change this to Bookmark was added succesfully

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -233,43 +233,43 @@
...
+    <string name="phone_permission">Phone: for getting your non-privacy Android device info to do simple statistics and collect error exception when FC.\n\n</string>
     <string name="force_cancel">Force cancel</string>
-    <string name="restore_tabs_hint">Restore the not-closed tabs last time?</string>
+    <string name="restore_tabs_hint">Restore the unclosed tabs last time?</string>
     <string name="restore_tabs">Restore Tabs on Startup</string>
-    <string name="install_success">Install Success!!!</string>
-    <string name="uninstall_success">Uninstall Success!!!</string>
-    <string name="online_addons">Online Addons</string>
+    <string name="install_success">Installation Successful!!!</string>
+    <string name="uninstall_success">Uninstallation Successful!!!</string>
```
[the remark this answers is not in the document]
Comment (zo-shin) [reply app/src/main/res/values/strings.xml]: Thanks for your attention. All the verb be is omitted to make the strings snappy.

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (zo-shin) [reply app/src/main/res/values/strings.xml]: Why should we spell it in all capital letters?

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (zo-shin) [reply app/src/main/res/values/strings.xml]: I don’t think the app is developed by or has anything to do with the website, so it’s not “its App”.

File: app/src/main/res/values/strings.xml (hunk 7 above)
[the remark this answers is not in the document]
Comment (Vv2233Bb) [reply app/src/main/res/values/strings.xml]: OK. I get it.:relaxed:

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (Vv2233Bb) [reply app/src/main/res/values/strings.xml]: AD is a typo on my end. I mean could you add an "an". It still sounds snappy.

Comment (zo-shin) [reply app/src/main/res/values/strings.xml]: Like the verb be, the article is always omitted, too. Hope you agree.

**Pull request merged**